os/pm/pm_initialize.c

```diff
@@ -131,6 +131,14 @@ void pm_initialize(void)
 		/* Add the initial state change node to the head of the history queue */
 
 		sq_addlast((&initnode->entry), &g_pmglobals.domain[domain_indx].history);
+#endif
+#ifdef CONFIG_IDLE_PM
+#if (CONFIG_ARCH_IDLE_PM_NSTATE > 0)
+
+		/* Initialize the sleep states' tick count threshold values */
+		pm_initialize_idle_data();
```

Comment:
No need of creating new file "pm_initialize_idle_data.c" 
keep the definition in this file itself.